server/app/services/WellKnownPaths.java

```diff
@@ -8,5 +8,5 @@ public class WellKnownPaths {
   public static Path APPLICANT_MIDDLE_NAME = Path.create("applicant.name.middle_name");
   public static Path APPLICANT_LAST_NAME = Path.create("applicant.name.last_name");
   public static Path APPLICANT_NAME = Path.create("applicant.name");
-  public static Path APPLICANT_DOB = Path.create("applicant.date_of_birth");
+  public static Path APPLICANT_DOB = Path.create("applicant.applicant.date_of_birth");
```

Comment:
Blocking: Why `applicant.applicant.date_of_birth`?

Isn't the path corresponding to the canonical question `applicant.applicant_date_of_birth`?

Canonical question definition: https://github.com/civiform/civiform/blob/16fc26e1bf0722fb29d253ecf137b8667d738528/server/app/tasks/DatabaseSeedTask.java#L77

If this is out of sync with the canonical question, it won't have the desired effect of auto-filling any applications that ask for the date of birth.